lib/PrintAsClang/DeclAndTypePrinter.cpp

```diff
@@ -1835,8 +1835,7 @@ class DeclAndTypePrinter::Implementation
 
   void visitExistentialType(ExistentialType *ET,
                             Optional<OptionalTypeKind> optionalKind) {
-    visitExistentialType(ET, optionalKind,
-        /*isMetatype=*/ET->getConstraintType()->is<AnyMetatypeType>());
+    visitPart(ET->getConstraintType(), optionalKind);
```

Comment:
This change is so that sugar is preserved when printing as ObjC, because now imported types can be `ExistentialType(TypeAliasType))`